UPGRADE.md

```diff
@@ -1,6 +1,17 @@
 eZ Platform Site API upgrade instructions
 =========================================
 
+Upgrading from 2.2 to 2.3
+-------------------------
```

Comment:
Needs to be 2.4 now :)

Reply:
That's an eagle eye @emodric, updated :)